Products/CMFPlomino/tests/samples.txt

```diff
@@ -91,6 +91,28 @@ Scenario::
     True
     >>> browser.open(base_url+'/basic-example/all')
 
+Test DateTime field::
+
+    >>> browser.open(base_url+'/basic-example')
+    >>> 'Birthday' in browser.contents
+    True
+    >>> browser.getLink('Birthday').click()
+    >>> browser.getControl(name='personName').value = 'Eric'
+    >>> browser.getControl(name='birthDate.year:record').value = ['1974']
+    >>> browser.getControl(name='birthDate.month:record').value = ['00']
+    >>> browser.getControl(name='birthDate.day:record').value = ['20']
+    >>> browser.getControl(name='personCategory').value = ['Friend']
+    >>> browser.getControl(name='plomino_save').click()
+    >>> '20-00-1974' in browser.contents
+    False
+    >>> browser.open(browser.url+"/EditDocument")
+    >>> browser.getControl(name='birthDate.year:record').value
+    ['0000']
+    >>> browser.getControl(name='birthDate.month:record').value
+    ['00']
+    >>> browser.getControl(name='birthDate.day:record').value
+    ['00']
+
```

Comment:
You need a test that tests what happens if a mantatory date is not touched. Does it give an error.
And also what happens if you edit a date and try and reset it. Does it delete the value, or at least do it what the old code used to do.